src/20250814_Multi_GHOInkLaunch/abstraction/constants/CCIPChainTokenAdminRegistries.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
```

Comment:
Add links for where to get these

Reply:
Done here: d862ce65618aa646cbda8b7ef6c0db43822052e4